### DISCUSSION TITLE
GitHub profile not loading on mobile app

### DISCUSSION THREAD

**Discussion by jasonsperske** [community/community#29107] [Mobile]
How would I debug why loading our company's GitHub repo (which works in iOS Safari https://github.com/grammarly/grammarly-for-developers) seems to break in the iOS GitHub client? ![github repo fails to load in iOS app](https://user-images.githubusercontent.com/125513/184062881-b5653e40-f655-4443-847f-93e305f68535.png)

The client is updated to the latest version (1.75.0)

![repo is working in iOS Safari](https://user-images.githubusercontent.com/125513/184062926-b865b934-8349-4995-b969-7cd7420afd39.png)

**Comment by othyn** [comment]
> How would I debug why loading our company's GitHub repo (which works in iOS Safari https://github.com/grammarly/grammarly-for-developers) seems to break in the iOS GitHub client? ![github repo fails to load in iOS app](https://user-images.githubusercontent.com/125513/184062881-b5653e40-f655-4443-847f-93e305f68535.png)
> 
> The client is updated to the latest version (1.75.0)
> 
> ![repo is working in iOS Safari](https://user-images.githubusercontent.com/125513/184062926-b865b934-8349-4995-b969-7cd7420afd39.png)

I'm having the same issue with the whole app, nothing works or loads. I've tried pretty much everything I can do; auth cycles, delete/re-download, cache clear, etc. nothing fixes it...

**Comment by alcere** [comment]
@jasonsperske thanks for reporting! I have created an issue, and the team will look at it soon! Thanks again for the details!
